example.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * This is a hypothetical implementation to understand
+ * what implementations may look like as we define
+ * a standard for video quality of experience metrics.
+ */
+
+// States
+var NOT_PLAYING = 0;
+var ATTEMPTING_TO_PLAY = 1;
+var PLAYING = 2;
+
+
+// Watch Time
+var watchTime = 0;
+var lastWatchTimeCheck;
+var watchTimeInterval;
+
+function updateWatchTime() {
+  watchTime = watchTime + (Date.now() - lastWatchTimeCheck);
```

Comment:
Yeah, that makes sense. With this I'm kind of aiming for something non-programmers can begin to read, but I think that fits.